events/mailer.go

```diff
@@ -39,6 +44,12 @@ func (d DelegatingEmailEventHandler) CanHandle(ce cloudevents.Event) bool {
 }
 
 func (d DelegatingEmailEventHandler) Handle(ce cloudevents.Event) error {
+	s, _ := json.MarshalIndent(ce.Type(), "", "\t")
```

Comment:
I don't think we should log here with `fmt.Println` as this might get too noisy without any ability to turn it off. I suggest using https://github.com/tidepool-org/go-common/blob/f7ecd77504bfe1fa3968dd3feae6e331efce48e3/events/events.go#L118-L129 for debugging purposes.

Reply:
Will remove. Accidentally left in place.